### ISSUE TITLE
[Feature] Option to disable k&m emulation of controller in games that support true k&m

### ISSUE THREAD

**Issue by Danfun64** [redphx/better-xcloud#391]
### Device

Phone/Tablet

### Operating System

Windows

### Browser

Chrome/Edge/Chromium

### Suggestion

Microsoft has recently added true keyboard and mouse support for a select number of games as a "preview feature". As of [this](https://news.xbox.com/en-us/2024/03/25/xbox-insider-release-notes-alpha-skip-ahead-2408-240321-2200/) Xbox Insider Release Notes page, 14 games can run K&M natively. It would be nice if Better XCloud could detect if the game being played supports native k&m support and bypass the emulated controller feature.

**Comment by redphx** [comment]
This feature has been implemented in 4.0.1.  
Which game did you test?

**Comment by Danfun64** [comment]
I didn't realize you implemented this feature. I should probably test a game before I close this though.

**Comment by Danfun64** [comment]
Okay, tested it out, noticed some issues.

- The ESC key doesn't seem to work when "Emulate Controller with Mouse and Keyboard" is turned on. (I noticed this when I tested Quake II)
- I'm not sure how I feel about the F8 key being hijacked by the enable/disable emulated mouse and keyboard functionality. It's probably no different from how F9 is hijacked for the Xbox menu, but I do find it worth noting.

I haven't tried to determine how real K&M affects local coop with one or more joysticks.
